packages/sdk-components-react/src/time.tsx

```diff
@@ -348,8 +348,18 @@ export const Time = React.forwardRef<React.ElementRef<"time">, TimeProps>(
       timeStyle: timeStyleOrUndefined(timeStyle),
     };
 
-    const datetimeString =
-      datetime === null ? INVALID_DATE_STRING : datetime.toString();
+    let datetimeString;
+    if (datetime === null) {
+      datetimeString = INVALID_DATE_STRING;
+    } else if (isNaN(Number(datetime)) === false) {
```

Comment:
datetime="2024.10" is valid Date but also a valid number

Reply:
Mb additionally check that 
`Number.isNaN(new Date().getTime(datetime)) && isNaN(Number(datetime)) === false`

Number.isNaN(new Date().getTime(datetime)) <-- means that Date is invalid

Reply:
Or check that parsed number is integer `Number.isInteger` and that it's above some maximum possible year like > 2100